css/style.css

```diff
@@ -0,0 +1,327 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+}
```

Comment:
Your project has a strange, horizontal scrolling behavior. The reason for that is that you didn't set your `margin` and `padding` to 0 here. I suggest you do so.

Additionally, your project has a margin that is not present in the template design.

Your project:

<img width="205" alt="Screen Shot 2022-05-26 at 8 33 10" src="https://user-images.githubusercontent.com/82426682/170498028-2dd1e06d-a082-4b24-8f1d-d155daa325ba.png">

The template design:

<img width="289" alt="Screen Shot 2022-05-26 at 8 33 39" src="https://user-images.githubusercontent.com/82426682/170498089-a2ed4807-1791-44e7-82cd-a8c77c9c9a81.png">

Kindly fix this.

Reply:
Done as suggested

Reply:
Done as advised